Assignment 3-JS/sass/style.scss

```diff
@@ -0,0 +1,235 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+html {
+  font-size: 80%;
+}
+.container-body {
+  width: 100%;
+  height: 100%;
+  background:linear-gradient(to left,rgb(163, 243, 226),rgb(45, 190, 190),rgb(2, 129, 129));
+  color: white;
```

Comment:
Move into a new file and use that here

Reply:
Moved it in another sass file and imported it here.